pkg/clusterapi/apibuilder_test.go

```diff
@@ -48,6 +51,7 @@ func (c *providerMachineTemplate) DeepCopyObject() runtime.Object {
 }
 
 func newApiBuilerTest(t *testing.T) apiBuilerTest {
+	format.MaxLength = 0
```

Comment:
What does this do?

Reply:
it prevents truncation of the objects being compared in unit tests. It helps debug the tests

Reply:
Can we remove it if it's not needed anymore then?

Reply:
i kinda want to keep it for all the apibuilder tests - it helps w troubleshooting in general

Reply:
Instead of `MaxLength`, you can swap `Equal` by `BeComparableTo`
That will print only the diff between the two objects as opposed to the whole structs. It makes it way easier to debug tests when comparing two big structs